xds/src/main/java/io/grpc/xds/internal/sds/trust/SdsX509TrustManager.java

```diff
@@ -223,15 +225,16 @@ void verifySubjectAltNameInChain(X509Certificate[] peerCertChain) throws Certifi
     if (certContext == null) {
       return;
     }
-    List<String> verifyList = certContext.getVerifySubjectAltNameList();
+    List<StringMatcher> verifyList = certContext.getMatchSubjectAltNamesList();
     if (verifyList.isEmpty()) {
       return;
     }
     if (peerCertChain == null || peerCertChain.length < 1) {
       throw new CertificateException("Peer certificate(s) missing");
     }
     // verify SANs only in the top cert (leaf cert)
-    verifySubjectAltNameInLeaf(peerCertChain[0], verifyList);
```

Comment:
Can you comment this instead of deleting the call? Since the comment above is still there.

Reply:
Done.